bootstrap.sh

```diff
@@ -52,14 +51,11 @@ else
   exit 1;
 fi
 
-which pip2 || curl -s https://bootstrap.pypa.io/get-pip.py | python2.7;
-pip2 install -U pip;
-which pip3 || curl -s https://bootstrap.pypa.io/get-pip.py | python3.6;
```

Comment:
get-pip is now the canonical method for installing pip according to the PyPA and ensures the user is always installing pacakges with the latest version of pip.

Reply:
Also, by removing this line, you would break the minishift env.